src/DependencyInjection/Facade.php

```diff
@@ -0,0 +1,38 @@
+<?php declare(strict_types=1);
+
+namespace Leadin\SurvivalKitBundle\DependencyInjection;
+
+use Psr\Container\ContainerInterface;
+
+abstract class Facade
+{
+    public const TAG        = 'survival_kit.facade';
```

Comment:
seems not other choice, but public const are so ugly ...

Reply:
agree